src/libraries/Microsoft.PowerFx.Core/Public/Types/AggregateType.cs

```diff
@@ -71,7 +71,13 @@ public virtual bool TryGetFieldType(string name, out FormulaType type)
         public bool TryGetFieldType(string displayOrLogicalName, out string logical, out FormulaType type)
         {
             Contracts.CheckNonEmpty(displayOrLogicalName, nameof(displayOrLogicalName));
-            if (_type.DisplayNameProvider.TryGetDisplayName(new DName(displayOrLogicalName), out _))
+
+            // checking null and empty case in-case derived types did not provide DisplayNameProvider.
+            if (_type.DisplayNameProvider == null || _type.DisplayNameProvider.LogicalToDisplayPairs.Count() == 0)
```

Comment:
>_type.DisplayNameProvider [](http://example.com/codeflow?start=16&length=25)

actually, I wonder - should we be spceial casing on DisplayName at all? 

Because unfortunately, there are other means of providing display names - Dtype has methods to handle all the cases. 

Can we call something like:
DType.TryGetDisplayNameForColumn